lib/pushnotify.js

```diff
@@ -22,22 +22,10 @@ function init(env, ctx) {
   var recentlySent = new NodeCache({ stdTTL: TIME_15_MINS_MS, checkperiod: 20 });
 
   pushnotify.emitNotification = function emitNotification (notify) {
-    if (!ctx.pushover) return;
-
     if (notify.clear) {
       console.info('got a notify clear');
-      var receiptKeys = receipts.keys();
-
-      _.forEach(receiptKeys, function eachKey (receipt) {
-        ctx.pushover.cancelWithReceipt(receipt, function cancelCallback (err, response) {
-          if (err) {
-            console.error('error canceling receipt, err: ', err);
-          } else {
-            console.info('got a receipt cancel response');
-          }
-        });
-        receipts.del(receipt);
-      });
+      if (ctx.pushover) cancelPushoverNotifications();
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '{' and instead saw 'cancelPushoverNotifications'.](https://www.codacy.com/app/jasoncalabrese/cgm-remote-monitor/file/758176619/issues/source?bid=2449627&fileBranchId=2449700#l27)